tests/queries/test_qs_combinators.py

```diff
@@ -89,6 +91,14 @@ def test_ordering(self):
         qs2 = Number.objects.filter(num__gte=2, num__lte=3)
         self.assertNumbersEqual(qs1.union(qs2).order_by('-num'), [3, 2, 1, 0])
 
+    def test_count(self):
+        qs1 = Number.objects.filter(num__lte=1)
+        qs2 = Number.objects.filter(num__gte=2, num__lte=3)
+        with CaptureQueriesContext(connection) as captured_queries:
+            self.assertEqual(qs1.union(qs2).count(), 4)
+        self.assertEqual(len(captured_queries), 1)
+        self.assertTrue(captured_queries[0]['sql'].upper().startswith("SELECT COUNT(*) FROM ("))
```

Comment:
As that fails on Oracle and might be brittle for third-party DBs, I think it could probably be removed?